### PR TITLE
MAUI-2853 [UG] Document for trackball behavior

### PR DESCRIPTION
**Pull request by Sowndharya1911**
Added UG document for trackball behavior

### CODE REVIEW

File: MAUI/Cartesian-Charts/Trackball.md [hunk 1]
```diff
@@ -0,0 +1,299 @@
...
+
+this.Content = chart;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+## Enable Label Display Mode
+
+The Label [DisplayMode]() property is used to specify whether to display a label for all the data points along the trackball line or display only the nearest data point label. The following are the options you can set for this property.
```
Comment (Saravanan-Madhesh) [inline MAUI/Cartesian-Charts/Trackball.md:53]: The `Label` [DisplayMode]()

Remove label

File: MAUI/Cartesian-Charts/Trackball.md [hunk 2]
```diff
@@ -0,0 +1,299 @@
...
+
+this.Content = chart;
+        
+{% endhighlight %}
+
+{% endtabs %}
+
+## Show or Hide the Series Label
+
+This feature is used to show or hide the trackball label of the series by using the [CartesianSeries.ShowTrackballLabel]() property. The default value is True.
```
Comment (Saravanan-Madhesh) [inline MAUI/Cartesian-Charts/Trackball.md:227]: The ShowTrackballLabel property can be used to hide the particular series trackball label. Its default value is true.

Comment (Sowndharya1911) [reply MAUI/Cartesian-Charts/Trackball.md]: Changed the content

File: MAUI/Cartesian-Charts/Trackball.md (hunk 1 above)
Comment (Sowndharya1911) [reply MAUI/Cartesian-Charts/Trackball.md]: removed the label

**Review by Saravanan-Madhesh — APPROVED** [review]
The changes are fine

**Pull request merged**